### ISSUE TITLE
Ghost NAs generated by ellipses in R dataframe display?

### ISSUE THREAD

**Issue by zazizoma** [ipython/ipython#9198]
R dataframe displays ellipses, and claims they are invalid factor levels to be replaced with NAs, which appear to be ghosts

[Dataframe issue.pdf](https://github.com/ipython/ipython/files/110265/Dataframe.issue.pdf)

![screen shot 2016-01-29 at 12 02 57 pm](https://cloud.githubusercontent.com/assets/10425738/12685280/aadb7864-c680-11e5-9e30-0eb97e8d1a9a.png)

**Comment by minrk** [comment]
This may be a question for the IR kernel: https://github.com/IRkernel/IRkernel

**Comment by takluyver** [comment]
It is, and if I'm understanding correctly, it's one that was already fixed. Make sure you update your versions of IRkernel, IRdisplay and repr.

**Comment by zazizoma** [comment]
Thanks Thomas, I tried a pip3 install --update IRkernel but got the message No matching distribution found for IRkernel. Would you mind pointing me towards instructions on how I would update these three files?

**Comment by takluyver** [comment]
IRkernel is not a Python package, so you don't upgrade it using pip. See
http://irkernel.github.io/ .

**Comment by zazizoma** [comment]
Hi 
so I went to the site recommended, ran from the terminal R console:
>update.packages(repos = c('http://irkernel.github.io/', getOption('repos')))

and then reattached with

 > IRkernel::installspec(user = FALSE).

I just updated jupyter today.

I closed everything, restared, and the notebook I sent behaves the same. I still get the warning/error message that the ellipses are trying to convert to NAs. They're not really, so I could ignore it and carry on, but they I'm never sure whether I've got NAs or not in any given situation.
